app/models/scheduled_message.rb

```diff
@@ -6,4 +6,16 @@ class ScheduledMessage < ActiveRecord::Base
   validates :title, presence: true
 
   acts_as_taggable
+
+  def self.filter(params)
+
+    results = self.all.where("lower(title) like ?", "%#{params[:title].downcase}%")
+    results = results.where("lower(body) like ?", "%#{params[:body].downcase}%")
```

Comment:
I find re-assignment like this can get kinda confusing sometimes. what about chaining instead:

```ruby
results = self.all.
  where("lower(title) like ?", "%#{params[:title].downcase}%").
  where("lower(body) like ?", "%#{params[:body].downcase}%")
```